src/mon/FSCommands.cc

```diff
@@ -374,70 +368,68 @@ class SetHandler : public FileSystemCommandHandler
 	ss << "enabled new snapshots";
       }
     } else if (var == "allow_multimds") {
-      bool enable_multimds = false;
-      int r = parse_bool(val, &enable_multimds, ss);
-      if (r != 0) {
-	return r;
-      }
-
-      if (!enable_multimds) {
-	fsmap.modify_filesystem(fs->fscid,
-	     [](std::shared_ptr<Filesystem> fs)
-		{
-		  fs->mds_map.clear_multimds_allowed();
-		});
-	ss << "disallowed increasing the cluster size past 1";
-      } else {
-        fsmap.modify_filesystem(
-            fs->fscid,
-            [](std::shared_ptr<Filesystem> fs)
-        {
-          fs->mds_map.set_multimds_allowed();
-        });
-	ss << "enabled creation of more than 1 active MDS";
-      }
+	   ss << "Multiple MDS is always enabled. Use the max_mds parameter to control the number of active MDSs allowed. This command is DEPRECATED and will be REMOVED from future releases.";
```

Comment:
Please hard-wrap this at 80 cols.

Reply:
I don't think error messages should be wrapped. That makes them ungreppable. CodingStyle doesn't specify this one way or the other, but I'd rather printed error messages all together.